plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -183,6 +200,16 @@ protected CachingHiveMetastore(HiveMetastore delegate, OptionalLong expiresAfter
         configValuesCache = buildCache(expiresAfterWriteMillis, refreshMills, executor, maximumSize, statsRecording, this::loadConfigValue);
     }
 
+    private static <K, V> LoadingCache<K, V> neverCache(Optional<Executor> executor, com.google.common.base.Function<K, V> loader)
```

Comment:
the `executor` (refresh executor) is useless for a "never cache", drop the param